www/pages/support.tsx

```diff
@@ -29,12 +29,35 @@ import TwitterSocialProof from '~/components/Sections/TwitterSocialProof'
 
 // Import Swiper styles if swiper used on page
 import 'swiper/swiper.min.css'
+import { useRouter } from 'next/router'
+import { NextSeo } from 'next-seo'
 
 type Props = {}
 
 const Index = ({}: Props) => {
+  // base path for images
+  const { basePath } = useRouter()
+
+  const meta_title = 'Support | Supabase'
```

Comment:
title: Help &amp; Support | Supabase
description: Find help and support for Supabase. Our support agents provide answers on all types of issues, including account information, billing, and refunds.

ps, the `&amp;` is for displaying a '&' character ;)